app/routes/bdb/components/BdbPage.tsx

```diff
@@ -106,21 +110,30 @@ const BdbPage = () => {
     },
     {
       title: 'Studentkontakt',
-      dataIndex: 'studentContact',
+      dataIndex: 'studentContacts',
       search: true,
       inlineFiltering: true,
-      filterMapping: (studentContact: UnknownUser) => {
-        if (studentContact && typeof studentContact === 'object') {
-          return studentContact.fullName;
+      filterMapping: (studentContacts: TransformedStudentCompanyContact[]) => {
+        if (studentContacts && typeof studentContacts === 'object') {
+          return studentContacts
+            .map((studentContact) => studentContact.user.fullName)
+            .join(' ');
         }
       },
+      render: (_, { studentContacts }) =>
+        studentContacts && (
+          <Flex column gap="var(--spacing-sm)">
+            {studentContacts.map((studentContact) => (
+              <UserLink user={studentContact.user} />
+            ))}
+          </Flex>
+        ),
     },
     {
       title: 'Notat',
       dataIndex: 'comment',
       centered: false,
       maxWidth: 200,
-      render: (_, company) => company.adminComment,
```

Comment:
Relevant change? Why not remove all the props?

Reply:
No, shouldn't have added this. This should be deleted in another PR.